src/Schema/TypeFormats/StringURI.php

```diff
@@ -12,6 +12,8 @@ class StringURI
     public function __invoke(string $value): bool
     {
         try {
+            // namespace 'League\Uri' is provided by multiple packages, but PHPStan does not support merging them
```

Comment:
looks suspicious. can you give me more details, what packages bring similar namespaces?